easybuild/easyconfigs/p/PANDAseq/PANDAseq-2.11-intel-2018a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'PANDAseq'
+version = '2.11'
+
+homepage = 'https://github.com/neufeld/pandaseq'
+description = """PANDASEQ is a program to align Illumina reads, optionally with PCR primers embedded in the sequence, and reconstruct an overlapping sequence."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+# download from https://github.com/neufeld/pandaseq/archive/v2.11.tar.gz
+source_urls = ['https://github.com/neufeld/%(namelower)s/archive/v%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+
+checksums = [('sha512', '1c37b8f777b698fe02df0248b9352c9550af7da0c739d0186f9973d4b4fcc30febc9c6345b098bd96ffb5dce703ccd201850fc543caad3e9da09b0e38fec4396')]
```

Comment:
Please use `SHA256` checksum, and drop the empty line above (pro tip: you can use `eb --inject-checksums` for this, cfr. https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html?#adding-or-replacing-checksums-using-inject-checksums